config.vim

```diff
@@ -202,7 +202,7 @@ set shiftwidth=4
 set nu
 
 " remove ugly vertical lines on window division
-set fillchars+=vert:\ 
+set fillchars+=vert:\
```

Comment:
this space is meaningful and needed, it's telling that the character to separate windows is a space